qiita_pet/templates/study_base.html

```diff
@@ -15,12 +16,26 @@
 
   function fill_artifact(aid) {
     $("#study-main").html("ARTIFACT " + aid + " HTML STUFF");
+    $.get("/admin/artifact/", {artifact_id: +aid })
```

Comment:
It's because javascript is very weakly typed, so it's better to make it explicit that an integer must be here.